src/index.js

```diff
@@ -8,7 +8,7 @@ ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
 );
 
```

Comment:
- [ ] OPTIONAL
Kindly ensure to remove unused imports.

**Error:**
![image](https://user-images.githubusercontent.com/12745474/155253678-a173662e-a69c-4b9f-bbb7-69ce19c6824a.png)